pages/database-management/logs.mdx

```diff
@@ -114,6 +114,23 @@ container:
 
 </Steps>
 
+## Hiding sensitive information (Enterprise)
+
+To enhance security, it's crucial to ensure that sensitive information like passwords is not logged. Here's an example of how you can mask passwords in your logs:

```

Comment:
Remove "like passwords". The immediate following example is sufficient.

Reply:
The phrase "you can mask..." implies that the user needs to take some action to turn on this feature. Is there a config setting?

Reply:
no its automatic